lib/core/lib/ops/userOps.js

```diff
@@ -456,7 +456,13 @@ const userOps = {
     })
     sqlQuery.append(SQL` ON CONFLICT ON CONSTRAINT user_policy_link DO NOTHING`)
 
-    client.query(sqlQuery, utils.boomErrorWrapper(cb))
+    client.query(sqlQuery, (err, result) => {
+      if (utils.isForeignKeyViolationError(err)) {
+        return cb(Boom.badRequest(`Invalid payload`))
```

Comment:
`Invalid payload` seems very generic, how about "some policies could not be found"?

BTW: I think that this problem should be fixed by checking policies and user belong to the same organization

https://github.com/nearform/labs-authorization/blob/23b033f8d07b0c8c8301ece72267ee7264ea6b2e/lib/core/lib/ops/userOps.js#L332-L335

I think the `utils.checkPoliciesOrg` is not working properly, otherwise it should give you an error (because of the invalid policy id)